image/templates/helm/stackrox-central/config/proxy-config.yaml.default

```diff
@@ -1,4 +1,4 @@
-# # NOTE: Both central and scanner should be restarted if this secret is changed.
+# # NOTE: Central, Scanner, and Scanner v4 Indexer should be restarted if this secret is changed.
```

Comment:
I don't think restart is actually required, but is rather here just in case. I think some of the original docs outdate the ability to periodically read the config. Similarly, it is possible there are components or `*http.Client`s which do not use a transport which can dynamically update. For consistency with the other StackRox components, I think we should keep the periodic updater and keep this doc